gzip.go

```diff
@@ -131,6 +133,18 @@ func (w *GzipResponseWriter) Flush() {
 	}
 }
 
+// Hijack implements http.Hijacker. If the underlying ResponseWriter is a
+// Hijacker, its Hijack method is returned. Otherwise an error is returned.
+func (w *GzipResponseWriter) Hijack() (net.Conn, *bufio.ReadWriter, error) {
+	if hj, ok := w.ResponseWriter.(http.Hijacker); ok {
+		return hj.Hijack()
+	}
+	return nil, nil, fmt.Errorf("Hijacker interface is not supported")
```

Comment:
It's valid here because `Hijacker` is the name of the interface to which the error is referring. Might be worth including the package name to avoid confusion.